workspace.json

```diff
@@ -345,6 +355,23 @@
             "buildTarget": "api:build"
           }
         },
+        "init-schema": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "parallel": false,
+            "commands": [
+              {
+                "command": "yarn ts-node -P apps/api/tsconfig.app.json apps/api/src/buildSchema.ts"
+              },
+              {
+                "command": "yarn nx run api:codegen"
+              },
+              {
+                "command": "yarn nx run web:codegen"
+              }
```

Comment:
Can web have its own `init-schema`? Since we'll end up with a lot of clients to this api.